README.md

```diff
@@ -1,2 +1,149 @@
+
+### Key Features <a name="key-features"></a>
+
+- **Rails API**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ChrispaixK/final-capstone-front-end.git
+```
+
+### Install
+
+#### 1. Navigate to the location of the folder in your machine:
+
+```
+you@your-Pc-name:~$ cd <final-capstone-front-end>
+```
+
+#### 2. Install the packages:
+
+```
+npm install
+```
+
+#### 3. Start the server:
+
+```
+npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Daniel Urimubenshi**
+
+- Github: [@benshidanny11](https://github.com/benshidanny11/)
+- LinkedIn: [@danielurimubenshi](https://www.linkedin.com/in/danielurimubenshi/)
+- Twitter: [@DBenshi](https://twitter.com/DBenshi)
+
+👤 **Rex**
+
+- Github: [@rex-9](https://github.com/rex-9/)
+- Medium: [@rex9](https://medium.com/rex9/)
+- LinkedIn: [@rex9](https://www.linkedin.com/in/rex9/)
+- Facebook: [@htetnaing0814](https://www.facebook.com/htetnaing0814)
+- Angelist: [@rex9](https://angel.co/u/rex9)
+
+👤 **Abraham Ibizugbe**
+
+- GitHub: [@ibizugbe](https://github.com/ibizugbe)
+- Twitter: [@abrahamibizugbe](https://twitter.com/abrahamibizugbe)
+- LinkedIn: [Abraham Ibizugbe](https://linkedin.com/in/abrahamibizugbe)
+
+👤 **Chrispaix Kaze**
+
+- GitHub: [@Chrispaix Kaze](https://github.com/ChrispaixK)
+- Twitter: [@Chrispaix Kaze](https://twitter.com/ChrispaixK)
+- LinkedIn: [Chrispaix Kaze](https://www.linkedin.com/in/chrispaix-kaze-70445a175/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please kindly offer me opportunities for further contributions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving the inspiration to build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I build and use this project on Windows**
+
+  - Yes, you can. Check my commit history. If it's hard to understand, you can easily reach out to me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪